### PR TITLE
[OTA - 1485] FE- installation events

### PR DESCRIPTION
**Pull request by JustynaKK**

### CODE REVIEW

File: ota-plus-web/app/reactapp/src/components/device/InstallationEvents.jsx [hunk 1]
```diff
@@ -1,124 +1,99 @@
...
+        events && events.map(el=> {
+            let time = moment(el.receivedAt).format("ddd MMM DD YYYY, h:mm:ss A");
+            switch (el.eventType.id) {
+                case 'EcuDownloadStarted':
+                    preparedEvents.ecuDownloadStarted.time= time;
+                    preparedEvents.ecuDownloadStarted.status = "Success";
+                    break;
+                case 'EcuDownloadCompleted':
+                    preparedEvents.ecuDownloadCompleted.time = time;
+                    el.payload.success? preparedEvents.ecuDownloadCompleted.status = "Success" : preparedEvents.ecuDownloadCompleted.status = 'Error';
```
Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/components/device/InstallationEvents.jsx]: `status = success ? "Success" : "Error"` ?

File: ota-plus-web/app/reactapp/src/components/device/history/MtuList.jsx [hunk 2]
```diff
@@ -29,10 +29,16 @@ class MtuList extends Component {
                 >
                     {packagesStore.packagesHistory.length ?
                         _.map(packagesStore.packagesHistory, (historyItem, index) => {
+                            let itemEvents = devicesStore.deviceEvents.filter(el=> {
+                               if ( el.payload.correlationId ) {
```
Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/components/device/history/MtuList.jsx]: Different whitespace style?

File: ota-plus-web/app/reactapp/src/components/device/queue/MtuList.jsx [hunk 3]
```diff
@@ -30,6 +30,11 @@ class MtuQueueList extends Component {
                         <ul className={"overview-panel__list" + (!devicesStore.multiTargetUpdates.length ? " empty" : "")}>
                             {_.map(devicesStore.multiTargetUpdates, (update, index) => {
                                 return _.map(update.targets, (target, serial) => {
+                                let itemEvents = devicesStore.deviceEvents.filter(el=> {
+                                    if ( el.payload.correlationId ) {
```
Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/components/device/queue/MtuList.jsx]: Whitespace style?

File: ota-plus-web/app/reactapp/src/components/device/history/MtuList.jsx [hunk 4]
```diff
@@ -29,10 +29,16 @@ class MtuList extends Component {
                 >
                     {packagesStore.packagesHistory.length ?
                         _.map(packagesStore.packagesHistory, (historyItem, index) => {
+                            let itemEvents = devicesStore.deviceEvents.filter(el=> {
```
Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/components/device/history/MtuList.jsx]: Whitespace? `e1 =>`

File: ota-plus-web/app/reactapp/src/components/device/InstallationEvents.jsx [hunk 5]
```diff
@@ -1,124 +1,99 @@
...
+                time: 'Pending',
+                status: 'Pending'
+            },
+        };
+
+        events && events.map(el=> {
+            let time = moment(el.receivedAt).format("ddd MMM DD YYYY, h:mm:ss A");
+            switch (el.eventType.id) {
+                case 'EcuDownloadStarted':
+                    preparedEvents.ecuDownloadStarted.time= time;
```
Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/components/device/InstallationEvents.jsx]: `time =` whitespace.

File: ota-plus-web/app/reactapp/src/components/device/queue/MtuList.jsx [hunk 6]
```diff
@@ -30,6 +30,11 @@ class MtuQueueList extends Component {
                         <ul className={"overview-panel__list" + (!devicesStore.multiTargetUpdates.length ? " empty" : "")}>
                             {_.map(devicesStore.multiTargetUpdates, (update, index) => {
                                 return _.map(update.targets, (target, serial) => {
+                                let itemEvents = devicesStore.deviceEvents.filter(el=> {
+                                    if ( el.payload.correlationId ) {
+                                        return el.payload.correlationId.search(update.updateId) >= 0
```
Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/components/device/queue/MtuList.jsx]: Where does it filter events for ECU serial? Will need to test against device with secondary ecu.

Comment (JustynaKK) [reply ota-plus-web/app/reactapp/src/components/device/queue/MtuList.jsx]: The test against secondary will be conducted when the info for secondary will be avaialble in `api/v1/auditor/update_reports` =>operationResults

**Pull request merged**